shared.bazelrc

```diff
@@ -27,6 +29,7 @@ build:workflows --remote_instance_name=buildbuddy-io/rules_xcodeproj/workflows
 build:workflows --color=yes
 build:workflows --terminal_columns=120
 build:workflows --disk_cache=
+build:workflows --@com_github_buildbuddy_io_rules_xcodeproj//xcodeproj:extra_generator_flags='--config=workflows --bes_backend= --bes_results_url='
```

Comment:
Since this only overrides `extra_generator_flags`, the fixtures will still have `--config=cache` in them.

@cgrindel @BalestraPatrick This means you need to use `--config=cache `when updating the fixtures. I'll update the guide.